plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GradleEnclavePlugin.kt

```diff
@@ -196,6 +196,8 @@ class GradleEnclavePlugin @Inject constructor(private val layout: ProjectLayout)
         return target.createTask("generateGramineManifest$type", type) { task ->
             task.pythonEnclave.set(pythonSourcePath != null)
             task.signingKey.set(signingKey)
+            task.isvProdId.set(conclaveExtension.productID)
+            task.isvSvn.set(conclaveExtension.revocationLevel)
```

Comment:
Yesterday, you showed me that the revocation level is set to isvSvn - 1. Shouldn't the value be incremented by one?

Reply:
The truth is that I can't pass that value here for some reason, being this a property. I increment this in the manifest

Reply:
`revocationLevel` is isvSvn + 1, either do that here, or rename `task.isvSvn` to `revocationLevel` and do it in the . I think the later is the correct way.